packages/front-end/components/GeneralSettings/FeaturesSettings.tsx

```diff
@@ -44,6 +46,20 @@ export default function FeaturesSettings() {
     );
   }, [form, codeRefsBranchesToFilterStr]);
 
+  useEffect(() => {
+    const requireReview = form.watch("requireReviews");
+    if (requireReview === false || requireReview === true) {
+      form.setValue("requireReviews", [
+        {
+          requireReview,
+          resetReviewOnChange: false,
+          environments: [],
+          projects: [],
+          tags: [],
+        },
+      ]);
+    }
+  }, [form]);
```

Comment:
This isn't the right place to do a migration.  It should be done in the back-end `migrations.ts`.  There's an `upgradeOrganizationDoc` method where we have migrations for several other setting fields that have changed in the past.